MAUI/PDF-Viewer/Migration.md

```diff
@@ -372,6 +372,24 @@ private void PdfViewer_PropertyChanged(object? sender, PropertyChangedEventArgs
 In .NET MAUI PDF Viewer, you can implement custom save logic at the application level. If you would like to include a Save button in the built-in toolbar, this can be achieved by customizing the toolbar items. Refer to the {{'[Customize toolbar items user guide](https://help.syncfusion.com/maui/pdf-viewer/toolbar#customize-toolbar-items)'| markdownify }} for instructions on adding a custom Save button.
 </td>
 </tr>
+<tr>
+<td>
+<div>{{'[CanUndoModified](https://help.syncfusion.com/cr/xamarin/Syncfusion.SfPdfViewer.XForms.SfPdfViewer.html#Syncfusion_SfPdfViewer_XForms_SfPdfViewer_CanUndoModified)'| markdownify }},</div>
+<div>{{'[CanRedoModified](https://help.syncfusion.com/cr/xamarin/Syncfusion.SfPdfViewer.XForms.SfPdfViewer.html#Syncfusion_SfPdfViewer_XForms_SfPdfViewer_CanRedoModified)'| markdownify }},</div>
+<div>{{'[CanRedoInkModified](https://help.syncfusion.com/cr/xamarin/Syncfusion.SfPdfViewer.XForms.SfPdfViewer.html#Syncfusion_SfPdfViewer_XForms_SfPdfViewer_CanRedoInkModified)'| markdownify }},</div>
+<div>{{'[CanUndoInkModified](https://help.syncfusion.com/cr/xamarin/Syncfusion.SfPdfViewer.XForms.SfPdfViewer.html#Syncfusion_SfPdfViewer_XForms_SfPdfViewer_CanUndoInkModified)'| markdownify }},</div>
+</td>
+<td>API Unavailable</td>
+<td>In the .NET MAUI PDF Viewer, there is no public APIs to detect the enabled or disabled state of the Undo and Redo buttons, unlike in Xamarin.
+However, you can reliably detect changes made to the PDF document by using the annotation-related events provided. These events include:
```

Comment:
Here need to mention and frame like answering what is the relation between undo/redo enabling and annotation changes
<img width="481" height="125" alt="image" src="https://github.com/user-attachments/assets/86d07986-f3ac-41db-8512-c7dcefc65784" />